src/server/data/models/user.ts

```diff
@@ -30,6 +30,9 @@ export default function defineUser(sequelize: Sequelize.Sequelize): IUserModel {
       type: Sequelize.STRING(256),
       field: 'last_name',
     },
+    resetPasswordId: {
+      type: Sequelize.UUID,
```

Comment:
I'm not sure if sequelize transforms this to snake case, usually I add the `field: 'reset_password_id'` just to be sure and not have surprises.